schemas/panels/time-series/time-series.cue

```diff
@@ -19,6 +19,7 @@ import (
 
 #legend: {
 	position: "Bottom" | "Right"
+	mode?: "List" | "Table"
```

Comment:
Making this optional, so the addition of the table mode isn't a breaking change for consumers.